mlrun/projects/pipelines.py

```diff
@@ -754,6 +754,7 @@ def _prepare_load_and_run_function(
         workflow_spec: WorkflowSpec,
         artifact_path: str,
         workflow_handler: str,
+        subpath: str,
         namespace: str,
```

Comment:
When adding new params to existing function we should try to avoid adding new parameters to function in the middle of the param list because this can break backwards compatibility. In this case we are ok because this is a private method which the user shouldn't access, but I would suggest verifying that all calls to that function aren't affected and changed.

Reply:
I can change it and move it to be the last parameter, i think it will be the easiest check

Reply:
so lets move it to be the last parameter (also in the function docstring)